### ISSUE TITLE
No loss reduction

### ISSUE THREAD

**Issue by laoxihongshi** [zhunzhong07/HHL#7]
thanks for your shared works，
I use torch0.4，
when i run baseline，i can get：
Mean AP: 19.9%
CMC Scores
  top-1          47.6%
  top-5          65.8%
  top-10         73.3%
  top-20         79.8%
but when i run hhl，the loss always is 6.735，
can you help me？

**Comment by zhunzhong07** [comment]
Could you provide the details of the parameters used for HHL? Maybe you have changed the batch size?

**Comment by laoxihongshi** [comment]
> Could you provide the details of the parameters used for HHL? Maybe you have changed the batch size?

yeah, I only set batch size = 120.
does it have to be equal to 128 to  converge?

**Comment by zhunzhong07** [comment]
Could you try the batch-size = 128? But I think that changing the batch size would not influence the training of HHL. Could you provide more details that you changed?

**Comment by NothingToSay99** [comment]
I also have the same problem that the loss(always 6.73) don`t decline when running hhl.

**Comment by zhunzhong07** [comment]
I have fixed the bug for pytorch>0.4.0. Please check the latest code.
